### PR TITLE
Nerfs pre-loaded pacman plasma sheets

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Lowers the number of plasma sheets in pre-loaded PACMAN generators from 50 to 15.

## Why It's Good For The Game

Being able to run around looting the generators and selling it to make like 10k round-start is not so bueno.

## Changelog
:cl:
balance: Pre-loaded PACMAN generators now have 15 plasma sheets, instead of 50.
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
> Tldr: Cargo has an abundance system that got rid of the mat selling meta a long time ago when it gave you 50k credits each sale. Now each time you do it it should not give you squat. Maybe buff that instead? Or snowflake mat removal here.

Plasma has a price elasticity of 0.

**Comment by Shadowflame909** [comment]
Well. I still think it'd be a better option then making the make-shift alternative to engineering not doing their job useless because cargo sold everything not bolted down

**Comment by Redrover1760** [comment]
> Two things here. The first is that pretty sure a long time ago someone added a system that lowers the reward every time you sell an ore. Do it enough times and you get squat
> 
> Secondly, the sheets in those pacmans are calculated to give you enough time to do whatever you have to do in the midst of power running out because engineering isn't doing their job. It's like nerfing ghetto surgery or building with iron sheets. Your just kicking a player when they're already down.
> 
> Instead what needs to be done is the prevention of being able to remove sheets from these generators. I would say you could also do a snowflake buff of how much power they generate to keep it the same. But that'd go against pac man game design so don't do that.
> 
> Tldr: Cargo has an abundance system that got rid of the mat selling meta a long time ago when it gave you 50k credits each sale. Now each time you do it it should not give you squat. Maybe buff that instead? Or snowflake mat removal here.

What ACTUALLY needs to be done other than SNOWFLAKING is to reduce how much plasma is used.

If you need 50 full sheets to run a PACMAN for long enough to do what you need to do there is a serious problem with the PACMAN system. I mean, 50 sheets of plasma is literally insane and takes forever to mine, especially without real mining tech.

**Comment by JohnFulpWillard** [comment]
15 sheets of plasma would be enough to power a techfab long enough to get more plasma, at least. 50 is way too much plasma and turns PACMAN into free loot.
